etcdctlv3/command/global.go

```diff
@@ -37,13 +37,19 @@ func mustClient(cmd *cobra.Command) *clientv3.Client {
 	var cfgtls *transport.TLSInfo
 	tls := transport.TLSInfo{}
 	if tls.CertFile, err = cmd.Flags().GetString("cert"); err == nil {
-		cfgtls = &tls
+		if tls.CertFile != "" {
```

Comment:
/cc @heyitsanthony We should only set tls if the flag is not empty string.

I found there is another issue with current client library. We probably should let client library to create the initial conn, not accept one. Our client library should control how to create the grpc conn (tls or not, since it is in the config).